### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by arvmarandi**
Here is my initial submission.

### CODE REVIEW

**Comment by arvmarandi** [comment]
Here is my initial submission.

File: work/Arvand/Exercise1.3.png
Comment (Alvaro8gb) [inline work/Arvand/Exercise1.3.png:1]: I cant see any zombie

File: work/Arvand/daemon.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pid = os.fork()
+
+if pid > 0:
+    sys.exit(f"Parent {os.getpid()} (Child: {pid})")
+
+elif pid == 0:
+    print(f"Child {os.getpid()} (Parent: {os.getppid()})")
+
+    # File path to the Daemon function
+    with open("daemon_function.py", "r") as file:
```
Comment (Alvaro8gb) [inline work/Arvand/daemon.py:14]: good

File: work/Arvand/killer.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import os
+import signal
+
+# Fork a child process
+pid = os.fork()
+
+# pid greater than 0 indicates the parent process
+if pid > 0:
+    print(f"parent process with PID: {os.getpid()}, my child's PID is: {pid}")
```
Comment (Alvaro8gb) [inline work/Arvand/killer.py:9]: good

File: work/Arvand/Exercise1.2.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import sys
+import os
+
+pid = os.fork()
+
```
Comment (Alvaro8gb) [inline work/Arvand/Exercise1.2.py:6]: system is high level fuction using exec is best

File: work/Arvand/Exercise1.1.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import time
+import os
+
+# Fork a child process
+pid = os.fork()
+
```
Comment (Alvaro8gb) [inline work/Arvand/Exercise1.1.py:6]: the parents shouldd wait

**Review by Alvaro8gb — CHANGES_REQUESTED** [review]
Ist okey but you can improve

**Pull request closed without merging**